docs/python/testing.md

```diff
@@ -275,40 +257,49 @@ For example, the `test_decrement` functions given earlier are failing because th
+1. Save the file and run the tests again to confirm that they pass, and see that the gutter decorations also indicate passing status.
 
     > **Note**: running or debugging a test does not automatically save the test file. Always be sure to save changes to a test before running it, otherwise you'll likely be confused by the results because they still reflect the previous version of the file!
 
-The **Python: Debug All Tests** and **Python: Debug Test Method** commands (on both the Command Palette and Status Bar menu) launch the debugger for all tests and a single test method, respectively. You can also use the "bug" icons in **Test Explorer** to launch the debugger for all tests in a selected scope as well as all discovered tests.
+You can use the following commands from the Command Palette to debug tests:
+- **Test: Debug All Tests** - launches the debugger for all tests in your workspace.
+- **Test: Debug Tests in Current File** - launches the debugger for the tests you have defined in the file you have open in the editor.
+- **Test: Debug Test at Cursor** - launches the debugger only for the method where you have your cursor focused on the editor.  You can also use the "Debug Test" icons in **Test Explorer** to launch the debugger for all tests in a selected scope as well as all discovered tests.
 
-The debugger works the same for tests as for other Python code, including breakpoints, variable inspection, and so on. To customize settings for debugging tests, you can specify ```"request":"test"``` in the launch.json file in the ```.vscode``` folder from your workspace. This configuration will be used when you run **Python: Debug All Tests** and **Python: Debug Test Method** commands.
+The debugger works the same for tests as for other Python code, including breakpoints, variable inspection, and so on. To customize settings for debugging tests, you can specify ```"purpose": ["debug-test"]``` in the `launch.json` file in the ```.vscode``` folder from your workspace. This configuration will be used when you run **Test: Debug All Tests**, **Test: Debug Tests in Current File** and **Test: Debug Test at Cursor** commands.
 
 For example, the configuration below in the ```launch.json``` file disables the ```justMyCode``` setting for debugging tests:
 
 ```py
 {
-    "name": "Debug Tests",
+    "name": "Python: Current File",
     "type": "python",
-    "request": "test",
+    "request": "launch",
+    "program": "${file}",
+    "purpose": ["debug-test"],
     "console": "integratedTerminal",
     "justMyCode": false
 }
 ```
 
-If you have more than one configuration entry with ```"request":"test"```, the first definition will be used since we currently don't support multiple definitions for this request type.
+If you have more than one configuration entry with ```"purpose":"debug-test"```, the first definition will be used since we currently don't support multiple definitions for this request type.
```

Comment:
```suggestion
If you have more than one configuration entry with ```"purpose": ["debug-test"]```, the first definition will be used since we currently don't support multiple definitions for this request type.
```